renderer/next.config.js

```diff
@@ -0,0 +1,11 @@
+exports.webpack = config => Object.assign(config, {
+  target: 'electron-renderer',
+  devtool: false,
```

Comment:
We definitely want sourcemaps development. Could go with `devtool: 'cheap-module-source-map',` for perf.